src/Administration/Resources/app/administration/src/app/component/sidebar/sw-sidebar-item-filter/sw-sidebar-item-filter.html.twig

```diff
@@ -0,0 +1,78 @@
+{% block sw_product_list_sidebar_filter %}
+    <sw-sidebar-item
+            icon="default-action-filter"
+            title="Filter"
+            ref="filter">
+
+        <sw-container v-for="filterOption in filterOptions" :key="filterOption.name" columns="1fr " gap="6px"
+                      class="sidebar--filter-container">
+            <sw-switch-field
+                    v-if="filterOption.inputType === 'switch'"
+                    size="small"
+                    :label="filterOption.label"
+                    :placeholder="filterOption.placeholder"
+                    v-model="filter[filterOption.name]">
+            </sw-switch-field>
+
+            <sw-text-field
+                    v-else-if="filterOption.inputType === 'input'"
+                    :label="filterOption.label"
+                    :placeholder="filterOption.placeholder"
+                    :value="filter[filterOption.name]"
+                    @input="inputTrigger($event, filterOption.name)">
+            </sw-text-field>
+
+            <sw-number-field
+                    v-else-if="filterOption.inputType === 'number'"
+                    numberType="float"
+                    :label="filterOption.label"
+                    :placeholder="filterOption.placeholder"
+                    :value="filter[filterOption.name]"
+                    @change="inputTrigger($event, filterOption.name)">
+            </sw-number-field>
+
+            <sw-single-select
+                    v-else-if="filterOption.inputType === 'singleSelect'"
+                    v-model="filter[filterOption.name]"
+                    :label="filterOption.label"
+                    :placeholder="filterOption.placeholder"
+                    :options="filterOption.options"
+                    valueProperty="value"
+                    labelProperty="name"
+                    :value="filter[filterOption.name]">
+            </sw-single-select>
+
+            <sw-multi-select
```

Comment:
try to use `sw-entity-single-select` and `sw-entity-multi-select`. These components are designed to this kind of case